python/dgl/nn/pytorch/explain/subgraphx.py

```diff
@@ -91,7 +91,7 @@ def __init__(
         model,
         num_hops,
         coef=10.0,
-        high2low=False,
+        high2low=True,
```

Comment:
You also need to change the parameter description above.

Reply:
fixed.